srcd/superset/assets/src/uast/codemirror_override.less

```diff
@@ -316,3 +359,7 @@ http://ethanschoonover.com/solarized/img/solarized-palette.png
 .cm-s-solarized.cm-s-light .CodeMirror-activeline-background {
   background: rgba(0, 0, 0, 0.06);
 }
+
+.CodeMirror-linenumber {
+  background: ;
```

Comment:
what's the purpose of this empty declaration of the `.CodeMirror-linenumber` `background`?

Reply:
Solved. Thanks!